### DISCUSSION TITLE
Accept strictly currency (number) values between mui and rff

### DISCUSSION THREAD

**Discussion by joannajjliu** [lookfirst/mui-rff#524] [Q&A]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
- accept strictly currency (number) values between mui and rff
- Format currency correctly based on currency type, with correct separators (ex. commas)
- Achieve something/integrate with [react-number-format](https://www.npmjs.com/package/react-number-format) if possible?

**Describe alternatives you've considered**

**Additional context**

**Comment by lookfirst** [comment]
Thanks for your issue report. I'm not sure I understand the problem here. Maybe you can provide an example?
